packages/migration/src/migrations/v0031_normalizeResourceSizes.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function normalizeResourceSizes({ pages, ...rest }) {
```

Comment:
Do we need docs? All migrations take a story as input by convention?

Reply:
There's some documentation in the package's README.

Extending that is out of scope of this PR though, as this just adds another migration like we do regularly.